rel-eng/tito.props

```diff
@@ -866,6 +866,7 @@ whitelist = nodejs-babel-plugin-module-resolver
   rubygem-azure_mgmt_network
   rubygem-azure_mgmt_resources
   rubygem-azure_mgmt_storage
+  rubygem-azure_mgmt_subscriptions
```

Comment:
Please add it to the foreman-nightly-el8 whitelist section as well so that this is available on EL8.

Reply:
Should I just add `rubygem-azure_mgmt_subscriptions` or all the `rubygem-azure_mgmt_*` as I can't see any of those in the `foreman-nightly-el8` section?

Reply:
Ohh sorry, I meant add to the `foreman-plugins-el8` section! I think there you will fin dthe rest of the azure packages.

Reply:
done.